policies/queries/ecs/cluster_container_instances_unused.sql

```diff
@@ -0,0 +1,17 @@
+insert into aws_policy_results
+with attachment as (select distinct cluster_container_instance_cq_id
+                    from aws_ecs_cluster_container_instance_attachments)
+with cluster as (select cq_id, account_id from aws_ecs_clusters)
+select :'execution_time'                                 as execution_time,
+       :'framework'                                      as framework,
+       :'check_id'                                       as check_id,
+       'Unused ECS cluster insstances should be removed' as title,
```

Comment:
How can a cluster instance be unused?

Reply:
right, misread the doc

Reply:
So shouldn't this entire query be removed?

Reply:
already